[13.0][MIG] mgmtsystem_info_security_manual

flachica opened the pull request:
Migration to 13.0

Commit:
[REN] Rename module and fix formatting

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Rename manifest files

dreispt reviewed (changes requested):
If you rename the module, we should provide the migration to allow people an easy upgrade.
Let me know if you prefer to keep the original name or need help on the mig script.

flachica commented:
> If you rename the module, we should provide the migration to allow people an easy upgrade.
> Let me know if you prefer to keep the original name or need help on the mig script.

I need help on the mig script. I think that the correct name is mgmtsystem_information_security. I prefer rename

FernandoRomera commented:
View a example in https://apps.odoo.com/apps/modules/12.0/web_pwa_oca/

In case you previously installed web_pwa, run the following steps with odoo shell, after having installed openupgradelib:

> from openupgradelib import openupgrade
> openupgrade.update_module_names(env.cr, [('web_pwa', 'web_pwa_oca')], merge_modules=False)
> env.cr.commit()

flachica commented:
> n [https://apps.odoo.com/apps/modules/12.0/web_pwa_oca/](url):

The link is broken?

FernandoRomera commented:
Or this
https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_base/migrations/12.0.1.0.1/pre-migration.py

Commit:
[IMP] mgmtsystem_info_security_manual: black, isort

Commit:
Migration to 13.0 mgmtsystem_information_security

Renamed mgmtsystem_info_security_manual to mgmtsystem_information_security for compatibility with mgmtsystem module

flachica commented:
@FernandoRomera I include the script. Thanks

FernandoRomera commented on mgmtsystem_information_security/__manifest__.py:
```diff
@@ -0,0 +1,31 @@
+    "data": ["data/information_security_manual.xml"],
+    "demo": [],
+    "installable": True,
+}
```
Missing:
 "external_dependencies": {"python": ["openupgradelib"]},

And in requirements.txt:
openupgradelib

flachica replied:
Thanks for revision. Included yer

ivantodorovich commented on mgmtsystem_information_security/__manifest__.py:
```diff
@@ -0,0 +1,32 @@
+    "name": "Information Security Management System Manual",
+    "version": "13.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
```
```suggestion
    "website": "https://github.com/OCA/management-system",
```

ivantodorovich commented on mgmtsystem_information_security/migrations/13.0.1.0.0/pre-migration.py:
```diff
@@ -0,0 +1,12 @@
+        env.cr,
+        [("mgmtsystem_info_security_manual", "mgmtsystem_information_security")],
+        merge_modules=True,
+    )
```
This won't work, I think..

On current deployments `mgmtsystem_information_security` won't be installed, so this script will never run.

I'm not sure what the process is to handle this, most probably a script in https://github.com/OCA/openugprade ?

in any case, is it really worth the trouble to rename the module?

flachica replied:
All right. I will rename ASAP. More simple is more good. Thanks @ivantodorovich

flachica replied:
Renamed! Thanks

FernandoRomera commented:
@dreispt can you remove the request or review?

dreispt commented:
/ocabot merge nobump
